### PR TITLE
Make versions tag based and release on GitHub

### PR DESCRIPTION
**Pull request by firecyberice**
This publishes to the Docker hub and GH releases only on tags.

### CODE REVIEW

**Comment by StefanScherer** [comment]
ping @firecyberice what is the status of this PR? Should this be merged?

**Comment by StefanScherer** [comment]
ping @firecyberice

**Comment by firecyberice** [comment]
I' ll have a closer look into the code today.

File: .drone.yml [hunk 1]
```diff
@@ -51,21 +42,23 @@ publish:
...
     email: $$DOCKER_EMAIL
     repo: hypriot/rpi-swarm
+    file: Dockerfile
     tag:
-      - 1.2.2
-      - latest
+      - $$TAG
+    when:
+      event: tag
     save:
```
Comment (firecyberice) [inline .drone.yml]: I think this should be removed and the push to gh-releases as well. L50-52 and L57

File: .drone.yml [hunk 2]
```diff
@@ -51,21 +42,19 @@ publish:
...
     email: $$DOCKER_EMAIL
     repo: hypriot/rpi-swarm
+    file: Dockerfile
     tag:
-      - 1.2.3
-      - latest
-    save:
-      file: /drone/src/buildresult/latest/swarm.tar
+      - $$TAG
+    when:
```
Comment (StefanScherer) [inline .drone.yml]: why not tagging latest as well?
A new tag build should update latest as well on Docker Hub. WDYT?

**Comment by StefanScherer** [comment]
We no longer have drone armbuilder

**Pull request closed without merging**